package.json

```diff
@@ -10,12 +10,10 @@
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --concurrency=1",
```

Comment:
new version of lerna seems to barf every couple of builds with an error about a callback not being called.  Setting concurrency to 1 slows it down but is more stable.